girder/girder_large_image/web_client/vue/components/PresetsMenu.vue

```diff
@@ -123,6 +146,16 @@ export default {
                     this.$emit('setCurrentFrame', preset.frame)
                 }
                 if (preset.style && Object.keys(preset.style.bands).length) {
+                    preset.style.bands = Object.fromEntries(
```

Comment:
This converts bands into a dictionary, but it needs to be an array.

Reply:
Is [f26394c](https://github.com/girder/large_image/pull/1248/commits/f26394c28c502314d0b7f54b29d06408d260a4a1) a sufficient fix?

Reply:
Yes, it appears to be.